_schema.graphql

```diff
@@ -2696,1341 +2710,187 @@ type BuyOrder implements Order {
-  SELLER
-}
-
-# Represents either a partner or a user
-union CommerceOrderPartyUnion = CommercePartner | CommerceUser
-
-enum CommerceOrderStateEnum {
-  # order is abandoned by buyer and never submitted
-  ABANDONED
-
-  # order is approved by seller
-  APPROVED
-
-  # order is canceled
-  CANCELED
-
-  # order is fulfilled by seller
-  FULFILLED
-
-  # order is still pending submission by buyer
-  PENDING
-
-  # order is refunded after being approved or fulfilled
-  REFUNDED
-
-  # order is submitted by buyer
-  SUBMITTED
-}
-
-# An error response for changes to an order
-type CommerceOrderWithMutationFailure {
-  error: CommerceApplicationError!
-}
-
-# A successfully returned order type
-type CommerceOrderWithMutationSuccess {
-  order: CommerceOrder!
-}
-
-type CommercePageCursor {
-  # first cursor on the page
-  cursor: String!
-
-  # is this the current page?
-  isCurrent: Boolean!
-
-  # page number out of totalPages
-  page: Int!
-}
-
-type CommercePageCursors {
-  around: [CommercePageCursor!]!
-
-  # optional, may be included in field around
-  first: CommercePageCursor
-
-  # optional, may be included in field around
-  last: CommercePageCursor
-  previous: CommercePageCursor
-}
-
-# Information about pagination in a connection.
-type CommercePageInfo {
-  # When paginating forwards, the cursor to continue.
-  endCursor: String
-
-  # When paginating forwards, are there more items?
-  hasNextPage: Boolean!
-
-  # When paginating backwards, are there more items?
-  hasPreviousPage: Boolean!
-
-  # When paginating backwards, the cursor to continue.
-  startCursor: String
-}
-
-type CommercePartner {
-  id: String!
-  type: String!
-}
-
-type CommercePickup {
-  fulfillmentType: String!
-}
-
-# Autogenerated input type of RejectOrder
-input CommerceRejectOrderInput {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
-  id: ID!
-}
-
-# Autogenerated return type of RejectOrder
-type CommerceRejectOrderPayload {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
-
-  # A union of success/failure
-  orderOrError: CommerceOrderOrFailureUnion!
-}
+    convert_to_utc: Boolean
+    format: String
 
-# Represents either a shipping information or pickup
-union CommerceRequestedFulfillmentUnion = CommercePickup | CommerceShip
+    # A tz database time zone, otherwise falls back to `X-TIMEZONE` header
+    timezone: String
+  ): String
+  stateUpdatedAt(
+    # This arg is deprecated, use timezone instead
+    convert_to_utc: Boolean
+    format: String
 
-# Autogenerated input type of SellerAcceptOffer
-input CommerceSellerAcceptOfferInput {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
-  offerId: ID!
-}
+    # A tz database time zone, otherwise falls back to `X-TIMEZONE` header
+    timezone: String
+  ): String
+  stateExpiresAt(
+    # This arg is deprecated, use timezone instead
+    convert_to_utc: Boolean
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 2729;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `convert_to_utc`